docs/tutorials/UKFast_SafeDNS.md

```diff
@@ -0,0 +1,208 @@
+# Setting up ExternalDNS for Services on UKFast's SafeDNS
+
+This tutorial describes how to setup ExternalDNS for usage within a Kubernetes cluster using SafeDNS
+
+Make sure to use **>=FIXME** version of ExternalDNS for this tutorial.
+
+## Managing DNS with SafeDNS
+
+If you want to learn about how to use the SafeDNS service read the following tutorials:
+To learn more about the use of SafeDNS in general, see the following page:
+
+[UKFast's SafeDNS documentation](https://docs.ukfast.co.uk/domains/safedns/index.html)
+
+## Creating SafeDNS credentials
+
+Generate a fresh API token for use with ExternalDNS, following the instructions
+at the UKFast developer [Getting-Started](https://developers.ukfast.io/getting-started)
+page. You will need to grant read/write access to the SafeDNS API. No access to
+any other UKFast service is required.
+
+The environment variable `SAFEDNS_TOKEN` must have a value of this token to run
+ExternalDNS with SafeDNS integration.
+
+## Deploy ExternalDNS
+
+Connect your `kubectl` client to the cluster you want to test ExternalDNS with.
+Then apply one of the following manifests file to deploy ExternalDNS.
+
+### Manifest (for clusters without RBAC enabled)
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        image: k8s.gcr.io/external-dns/external-dns:FIXME
```

Comment:
This needs edit too. Please add a comment that says that folks need to find the last version by looking at the GitHub releases as we currently still don’t have a process to replace all occurrences of the version in manifests in the doc.